build.gradle

```diff
@@ -123,6 +123,7 @@ subprojects {
         // TODO audit and fix our javadocs so that we don't need this setting
         // This is mainly for cases where param/throws tags don't have descriptions
         // Previously, javadocs weren't being compiled, but now shipkit automatically enables this build step
+        options.addStringOption('Xdoclint:none', '-quiet') // Suppress lint warnings
```

Comment:
thanks for finding this option!

Reply:
I couldn't find a way to really customize it and error out on specific things, it's basically all or nothing so I chose nothing to cut down the noise.